### DISCUSSION TITLE
Settings screen improvements - scroll, web links to github w/out login.

### DISCUSSION THREAD

**Discussion by mars4science** [PaulWoitaschek/Voice#1711] [Ideas]
I had large screen setting on Android and had no idea the Settings screen in app has version number at the bottom as page ended with another line. Please add scrolling or something to be able to see all of the page.

Links "Support, Suggest, " somehow on my phone resulted in github sign up/in page. I'm able to open this project on the phone w/out login in in browser from web search. Can you please make links to the page to work w/out login?

**Comment by PaulWoitaschek** [comment]
Moved to a bug:
https://github.com/PaulWoitaschek/Voice/issues/1713
